llvm/tools/llvm-readobj/ELFDumper.cpp

```diff
@@ -7365,6 +7369,55 @@ template <class ELFT> void LLVMELFDumper<ELFT>::printDynamicTable() {
   W.startLine() << "]\n";
 }
 
+template <class ELFT>
+void JSONELFDumper<ELFT>::printAuxillaryDynamicTableEntryInfo(
+    const Elf_Dyn &Entry) {
+  switch (Entry.getTag()) {
+  case DT_SONAME:
+    this->W.printString("Name", this->getDynamicString(Entry.getVal()));
+    break;
+  case DT_AUXILIARY:
+    LLVM_FALLTHROUGH;
```

Comment:
I'm not sure `LLVM_FALLTHROUGH` is needed where there's no body to the case statement. You should be able to just do:
```
case DT_AUXILIARY:
case DT_FILTER:
case DT_NEEDED:
  ListScope L(this->W, "Libraries");
```

Reply:
changed